jupyter_resource_usage/api.py

```diff
@@ -75,6 +75,20 @@ async def get(self):
 
             metrics.update(cpu_percent=cpu_percent, cpu_count=cpu_count)
 
+        # Optionally get Disk information
+        if config.track_disk_usage:
+            try:
+                disk_info = psutil.disk_usage(config.disk_path)
+            except:
+                pass
+            else:
+                metrics.update(disk_used=disk_info.used, disk_total=disk_info.total)
+                limits["disk"] = {"disk": disk_info.total}
+                if config.disk_warning_threshold != 0:
+                    limits["disk"]["warn"] = (disk_info.total - disk_info.used) < (
+                        disk_info.total * config.cpu_warning_threshold
```

Comment:
shouldn't this be 
```python
disk_info.total * config.disk_warning_threshold
```
??

Reply:
Indeed it should!

Reply:
Additionally, the warning state calculated is never handled. It looks like the current client-side code handles warning state for memory only. I'm playing with this right now to show the disk warning!

Reply:
More than happy for you to contribute....
Question: does the CPU have a parallel warning state that should be handled?

Reply:
Great question! I'm more than happy to contribute to your branch and close PR #233, which I've pulled in your changes to and did the React work for the warnings. I was in a flow state this AM and wasn't sure how active this PR was going to be, so I threw those changes together.

Check it out and let me know what you think!